scripts/quests/otherAreas/Bombs_Away.lua

```diff
@@ -62,38 +62,7 @@ quest.sections =
             {
                 [8] = function(player, csid, option, npc)
                     if quest:complete(player) then
-                        player:tradeComplete()
-                    end
-                end,
-            },
-        },
-    },
-    {
-        check = function(player, status, vars)
-            return status == QUEST_COMPLETED
-        end,
-
-        [xi.zone.ULEGUERAND_RANGE] =
-        {
-            ['Buffalostalker_Dodzbraz'] =
-            {
-                onTrigger = function(player, npc)
-                    return quest:event(7)
-                end,
-
-                onTrade = function(player, npc, trade)
-                    if npcUtil.tradeHasExactly(trade, { { xi.items.CLUSTER_CORE, 2 } }) then
-                        return quest:progressEvent(8)
-                    end
-                end,
-            },
-
-            onEventFinish =
-            {
-                [8] = function(player, csid, option, npc)
-                    player:delQuest(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.BOMBS_AWAY)
-                    if quest:complete(player) then
-                        player:tradeComplete()
+                        player:confirmTrade()
```

Comment:
is this confirmed to work? If so, looks good

Reply:
Yea, the latest commit works for me as expected (assuming you're not expected to reaccept the quest every time and can just trade him 2x every time after talking to him initially)

Reply:
I assume this is no longer completing the quest since you've removed that code? That means the quest will always be active for the player. I wouldn't want to do this unless that's expected behavior from retail. Otherwise might need to break this out into two sections, one that checks if available and one that checks if completed so that the one section can properly complete quest.

Reply:
It's checking if the quest is not available and falls into here on either accepted or completed. The quest completes as expected and I don't see any logs indicating that something bad happens if you complete a completed quest. The C++ looks like it's a no-op for quests that are already completed.

Although testing above I did find that there was another issue that you always accepted the quest despite saying you'd help or not. That's fixed now.

Reply:
I'm not sure why I thought I saw the `quest:complete(player)` was being deleted but looking back now its obviously not. So disregard!